webui/react/src/components/Spinner.test.tsx

```diff
@@ -0,0 +1,89 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import Spinner from './Spinner';
+
+describe('Spinner', () => {
+  loadAntdStyleSheet(); // defined in setupTests.ts
```

Comment:
if we end up going this route unloading the styles at the end of this test suite could be an option

Reply:
this happens automatically: loading in this test does not affect the other tests.